### ISSUE TITLE
[Newbee]Does paddle support second-order gradient?

### ISSUE THREAD

**Issue by Naruto-Sasuke** [PaddlePaddle/Paddle#12234]
I want to train a model with wgan-gp, which requires paddle supporting second-order gradient.
Also, spectral norm is also a feature that I need.

**Comment by reyoung** [comment]
Paddle does not support second-order gradient right now.

**Comment by lucywsq** [comment]
Hello, this issue has not been updated in the past month. we will close it today for the sake of other user’s experience. If you still need to follow up on this question after closing, please feel free to reopen it. In that case, we will get back to you within 24 hours. We apologize for the inconvenience caused by the closure and thank you so much for your support of PaddlePaddle Group!

**Comment by Naruto-Sasuke** [comment]
Ok, never mind, since paddle tends to be engineering preference, not scientistic preference. I don't think you guys have time to extend paddle to support second-order gradient which has been supported in pytorch more than one year ago!  Since, wgan-gp seems to perform no better than spectralNorm or relativelistic GAN,  I am free to let this closed.

**Comment by Joejiong** [comment]
> Ok, never mind, since paddle tends to be engineering preference, not scientistic preference. I don't think you guys have time to extend paddle to support second-order gradient which has been supported in pytorch more than one year ago! Since, wgan-gp seems to perform no better than spectralNorm or relativelistic GAN, I am free to let this closed.

hi, I think recently they have add paddle fluid api, which is a series of torch_like api, and they also support 2rd gradient
